src/pages/embed.vue

```diff
@@ -228,6 +240,31 @@ onBeforeMount(() => {
     window.logixProjectId = route.params.projectId
 })
 
+onMounted(() => {
+    switch (theme.value) {
+    case 'default':
+        updateThemeForStyle(THEME.default);
+        break;
+    case 'night-sky':
+        updateThemeForStyle(THEME.night_sky);
+        break;
+    case 'lite-born-spring':
+        updateThemeForStyle(THEME.lite_born_spring);
+        break;
+    case 'g-and-w':
+        updateThemeForStyle(THEME.g_and_w);
+        break;
+    case 'high-contrast':
+        updateThemeForStyle(THEME.high_contrast);
+        break;
+    case 'color-blind':
+        updateThemeForStyle(THEME.color_blind);
+        break;
```

Comment:
```suggestion
    case 'default':
        updateThemeForStyle(THEME.default);
        break;
    case 'night-sky':
        updateThemeForStyle(THEME.night_sky);
        break;
    case 'lite-born-spring':
        updateThemeForStyle(THEME.lite_born_spring);
        break;
    case 'g-and-w':
        updateThemeForStyle(THEME.g_and_w);
        break;
    case 'high-contrast':
        updateThemeForStyle(THEME.high_contrast);
        break;
    case 'color-blind':
        updateThemeForStyle(THEME.color_blind);
        break;
```
updateThemeForStyle(theme.value)

refactor this

Reply:
as `theme.value` is coming from params, we cannot directly put `theme.value` inside the `updateThemeForStyle` func, would like to discuss if there is any better way